# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1392621639 18000
#      Mon Feb 17 02:20:39 2014 -0500
# Branch stable
# Node ID 85d31344fb23725791bc687337295d47416b9c17
# Parent  3c1a28978599d65f6b06112d8bcbfce34e95d86a
3.8.1 release candidate 2

* configure.ac (OCTAVE_VERSION): Bump to 3.8.1-rc2.
(OCTAVE_MINOR_VERSION): Bump to 1-rc2.
(OCTAVE_RELEASE_DATE): Set to 2014-02-14.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,31 +14,31 @@ dnl Copyright (C) 1993-2013 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ### 
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.62])
-AC_INIT([GNU Octave], [3.8.0], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [3.8.1-rc2], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=3
 OCTAVE_MINOR_VERSION=8
-OCTAVE_PATCH_VERSION=0
+OCTAVE_PATCH_VERSION=1-rc2
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2013 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2013-12-27"
+OCTAVE_RELEASE_DATE="2013-02-17"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do that is
 ## with library versioning, but not all systems support that.
 ## NOTE: This macro will be removed in a future version of Octave.  If
 ## you insist on checking for features using a version number, use the
 ## OCTAVE_MAJOR_VERSION, OCTAVE_MINOR_VERSION, and
